docs/Developer-docs.md

```
@@ -71,6 +71,7 @@ format](https://github.com/angular/angular/blob/main/contributing-docs/commit-me
 | **test**     | Adding missing tests, correcting existing tests, or refactors tests   |
 | **work**     | An incremental change working towards a feature                       |
 
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)